lib/components/CustomYieldNotificationBanner.jsx

```diff
@@ -10,7 +10,7 @@ export const CustomYieldNotificationBanner = () => {
   if (prizePoolType !== PRIZE_POOL_TYPE.yield) return null
 
   return (
-    <NotificationBanner className='bg-purple-2'>
+    <NotificationBanner className='bg-purple-2' canClose>
```

Comment:
Another one I'd rather not let them be able to dismiss. This is important and people don't read, they just click X's.

I think it makes sense for users who are very competent: who would read that it's an unaudited yield source, know what that means and decide they want to stay and risk it, then close the warning. But very competent users are a bad gauge of what most need. 😉

Reply:
More of the same reasons as the other one. But I think it makes sense to force the tradeoff of space for the warning in this case.